### DISCUSSION TITLE
How to zoom out more than 100%?

### DISCUSSION THREAD

**Discussion by javedbartlett** [ValentinH/react-easy-crop#298] [Q&A]
Sometimes I have images that need to be zoomed out more than 100%. Is it possible to zoom out more than 100%?

If so, how would I do this? Could I just display a white background in the parts of the image that don't exist to match the aspect ratio?

**Comment by ValentinH** [comment]
Yes you can just use a `minZoom` smaller than 1.

**Reply by javedbartlett** [reply]
@ValentinH awesome, is there a way to move the image? I can zoom out more but the image is locked at the top left corner

**Reply by ValentinH** [reply]
Indeed this is a limitation. You could use `restrictPosition={false}` but it will remove all restrictions.
